public/posts/v1.6.md

```diff
@@ -0,0 +1,226 @@
+Today we are releasing Deno 1.6.0. This release contains some major features,
+and many bug fixes. Here are some highlights:
+
+- [**Build self contained, standalone binaries**](#codedeno-compilecode-building-self-contained-standalone-binaries):
+  `deno compile` can build your Deno projects into completly standalone
+  executables
+- [**Built-in Deno Language Server**](#built-in-deno-language-server): fully
+  integrated LSP for editor integration
```

Comment:
remove `integration`?

Reply:
agree - but probably better to just edit the text yourself.